src/transformers/models/videomae/image_processing_videomae.py

```diff
@@ -339,5 +339,5 @@ def preprocess(
             for video in videos
         ]
 
-        data = {"pixel_values": videos}
+        data = {"pixel_values": np.asarray(videos)}
```

Comment:
This looks great! We'll need to only apply this if `return_tensors` is not `None` this way:
* Behaviour is consistent with other processing classes: `return_tensors=None` returns a list of numpy arrays
* It won't break if the user hasn't asked for batching and the images/frames are of different sizes

```suggestion
        if return_tensors is not None: 
            # Speeds up tensor conversion - see: https://github.com/huggingface/transformers/pull/28221/files
            data = {"pixel_values": np.asarray(videos)}
```